### PR TITLE
SQL Fix Closes #104

### PR DESCRIPTION
**Pull request by gmross**
These are the fixes that I managed to pull together based on Owen's comments in #104 (SQL Fixes).  I believe this closes the issue(s) addressed.

### CODE REVIEW

**Review by r3cebarnett — DISMISSED** [review]

File: Software/scan_key/AuthenticationSQL.py [hunk 1]
```diff
@@ -1,42 +1,30 @@
...
-    cursor.execute("select machineID, machineType from MACHINE")
+    cursor.execute("SELECT machineType FROM MACHINE WHERE machineID = " + currMachineID)
+    
+    machineType = cursor.fetchone()
 
-    data1 = cursor.fetchall()
+    cursor.execute("SELECT auth1, auth2, auth3 FROM USER WHERE t1String = " + id) #getting user w/ the id    #might need to modify
 
-    machineType = None
+    authdata = cursor.fetchone()
```
Comment (ophilli) [inline Software/scan_key/AuthenticationSQL.py]: We don't want to download all three cols - just the one that is appropriate for our machine type.

File: Software/scan_key/AuthenticationSQL.py [hunk 2]
```diff
@@ -1,42 +1,30 @@
...
-    for origMachineID in data1:
-        if(origMachineID[0] == currMachineID[0]):
-            machineType = origMachineID[1]
-
-    cursor.execute("SELECT auth1, auth2, auth3 FROM USER WHERE t1String = " + id) #getting user w/ the id
-
-    authdata = cursor.fetchall()
-
-    for auth in authdata:
-        if(auth[machineType] != 1):
```
Comment (ophilli) [inline Software/scan_key/AuthenticationSQL.py:25]: The new line 17 should replace this logic.

**Review by ophilli — CHANGES_REQUESTED** [review]

**Comment by r3cebarnett** [comment]
Everything that I did can be seen in commit# 42577a7

Okay so I fixed how we get the stuffs from the SQL. There were a few things that were pretty screwy. So now we look up "authType" using "PiHostname" in the SQL, then look up the variable that authType is for ID in user. This is what is returned in machineAuth. For some reason, we had machineAuth returning the current machine, which makes no sense as we try to use it as a boolean later in the program but also as a machinetype in the SQL call in line 54, 55. This is like really stupid and needs to be fixed but I'm more concerned about fixing the algorithm than an event log. @gacosta, @gmross, and I will discuss a better way for us to do this. Also I'd like us to clear up a lot of the formatting for our code. It's very sporadic and all over the place where functions are. It'd be a lot of cleaning to do but I'd really like us to reorganize where we put functions. I found myself searching for a long time trying to find a specific function.

Anyways /rant. @ophilli does this fix the issue you had or do you have any further comments?

File: Software/scan_key/AuthenticationSQL.py [hunk 3]
```diff
@@ -8,16 +8,16 @@ def machineAuth(id, cursor):
...
     currMachineID = b(currMachineID).decode('UTF-8')
 
-    cursor.execute("SELECT machineType FROM MACHINE WHERE machineID = " + currMachineID)
+    cursor.execute("SELECT authType FROM MACHINE WHERE PiHostname = " + currMachineID)
     
-    machineType = cursor.fetchone()
+    authType = cursor.fetchone()
 
-    cursor.execute("SELECT auth1, auth2, auth3 FROM USER WHERE t1String = " + id) #getting user w/ the id    #might need to modify
+    cursor.execute("SELECT " + authType + " FROM USER WHERE t1String = " + id) #getting user w/ the id    #might need to modify
```
Comment (ophilli) [inline Software/scan_key/AuthenticationSQL.py]: t1String is not a column in the `catsadmin`.`users` table on watt.cuMaker.space.

Will accept once this issue is resolved. Might be resolved by adjusting code, or simply adding the column to the table. I'll leave this up to the software team to review.

**Review by ophilli — CHANGES_REQUESTED** [review]
Your changes look fine to me - except for the t1String issue.

**Review by ophilli — APPROVED** [review]

**Pull request merged**